ibm_catalog.json

```diff
@@ -1121,6 +1121,10 @@
                 {
                   "displayname": "4.14_openshift",
                   "value": "4.14_openshift"
+                },
+                {
+                  "displayname": "4.15_openshift",
+                  "value": "4.15_openshift"
```

Comment:
I think update the `default_value` to 4.15 so the tests will run it. We won't release it to prod for another month, so if there are issues, plenty of time to debug